foxtrot-core/src/main/java/com/flipkart/foxtrot/core/common/Action.java

```diff
@@ -239,4 +240,11 @@ protected Filter getDefaultTimeSpan() {
         return filters;
     }
 
+    public long getCountQueryTimeout() {
```

Comment:
Create one method which takes opcode as input and returns corresponding timeout.

Reply:
Does opcode make sense here as in ES query there is no significance of the same